structuredevent-model/src/main/java/com/sequenceiq/environment/parameter/dto/AzureResourceEncryptionParametersDto.java

```diff
@@ -0,0 +1,37 @@
+package com.sequenceiq.environment.parameter.dto;
+
+public class AzureResourceEncryptionParametersDto {
+    private final String keyUrl;
+
+    private AzureResourceEncryptionParametersDto(AzureResourceEncryptionParametersDto.Builder builder) {
```

Comment:
Please add unit tests for `AzureResourceEncryptionParametersDto` (at least test the builder & getter).